pkg/controller/operators/olm/operator.go

```diff
@@ -777,6 +779,18 @@ func (a *Operator) syncGcCsv(obj interface{}) (syncError error) {
 	return
 }
 
+func (a *Operator) recordMetrics(logger *logrus.Entry, csv *v1alpha1.ClusterServiceVersion) {
+	if csv.IsCopied() {
+		// don't record for copied CSVs
+		return
+	}
+	counter, err := metrics.CounterForCSV(csv.GetName(), string(csv.Status.Phase))
```

Comment:
isn't the CSV name unique for every version of the operator, that will make it hard for us to roll up counts that include all versions of an operator

i think what i would want would be the unique identifier for the operator as one label, the version is another, and then the phase

Reply:
We can report `Subscriptions` as well, labelled by package, which would give the identifier for which operators are installed sans version?

Reply:
I think that might be more aligned with our use of these metrics.

Reply:
so with subscriptions we don't know the currently installed version on that object do we, only sometimes the starting version? Is there a way (in followup work later) for us to report a status back to the subscription that says what is currently installed

not that we'd try to do that as part of this PR, just trying to brainstorm how we get to the end goal

Reply:
Subscriptions report the `currentCSV` in their status, but that doesn't give you information about the health of the CSV. 

I can switch to _only_ reporting metrics for subscriptions, with `name` and `currentCSV` labels?

Reply:
Just leaving this here for future reference; "we have a cluster_operator_conditions metrics that tells us if any CVO components are degraded.  is there an equivalent metric for tracking non CVO operators? something planned for OLM?"